documentation/samplefiles/readme.chsarp.md

```diff
@@ -10,6 +10,6 @@ csharp:
   payload-flattening-threshold: 1
```

Comment:
The name of the file is incorrect.  It should be 'readme.csharp.md'.  The rest of the changes look good.